### PR TITLE
update nginxinc/nginx-unprivileged:1.18-alpine sha & add hugo 0.75.1

### PR DESCRIPTION
**Pull request by ubergesundheit**
docs needs an upgrade. I want to use these container images.

### CODE REVIEW

File: images.yaml [hunk 1]
```diff
@@ -333,7 +333,7 @@
   tags:
   - sha: 2d55b56b04c06a4992729ad5430d2fae1a6833bd9c4a87a00173ff4563bbee40
     tag: 1.17-alpine
-  - sha: ae4131942b2e200dd06cbc52915b6972275ad56d1a37788661d65ead039d3cf3
+  - sha: 94da2c39b836871c1f15c49e150a5e7d9c8b24dcc5c8d6c557fee1300f2cbb0b
```
Comment (stone-z) [inline images.yaml:336]: Why are we moving tags?

Comment (ubergesundheit) [reply images.yaml:336]: its newer. But if this is not how we do things, we can use the old sha.

Comment (stone-z) [reply images.yaml:336]: I see, it looks like upstream moves tags? That is unfortunate. Then yeah, I guess this would be the most consistent with them.

Comment (ubergesundheit) [reply images.yaml:336]: So whats your opinion? Leave the sha alone or use the newer one? Could you approve?

**Review by stone-z — APPROVED** [review]
Someone asked for semver tags [here](https://github.com/nginxinc/docker-nginx-unprivileged/issues/48#issuecomment-694952836). Would be great if that happened but until then this is ok with me

**Comment by stone-z** [comment]
Someone asked for semver tags [here](https://github.com/nginxinc/docker-nginx-unprivileged/issues/48). Would be great if that happened but until then this is ok with me

**Pull request merged**